quintoandar_kafka/kafka_consumer.py

```diff
@@ -3,53 +3,55 @@
 
 from retrying import retry
 from kafka.consumer.group import KafkaConsumer
+from quintoandar_kafka import IdempotenceClient
+from functools import partial
 
 
-class KafkaConsumerClient:
-
-    def __init__(self, group_id, bootstrap_servers, topic,
-                 processor, deserializer=None, idempotence_client=None):
+class KafkaDefaultConsumer():
```

Comment:
That's ok but I think I prefer just `KafkaConsumer` (or possibly event `KafkaSimpleConsumer`, this has the bonus advantage of putting it after KafkaIdempotentConsumer on an alphabetical autocompletion list).

The `Default` suggests that this is the standard choice. People will import this lib and their first choice will be to use KafkaDefaultConsumer.